LullabyRecipe/Views/Kitchen/StudioView.swift

```diff
@@ -86,31 +98,91 @@ struct StudioView: View {
+                                
+                                selectedImageNames.natural = selectedNaturalSound.imageName
+                                
+                                opacityAnimateVars[2] = 0.5
                             }
                         }
                     case .melody:
                         RadioButtonGroup(selectedId: soundType.rawValue,
                                          items: melodySounds) { melodySounds in
                             selectedMelodySound = melodySounds
-
+                            
                             if selectedMelodySound.name == "Empty" {
                                 melodyAudioManager.stop()
+                                
+                                opacityAnimateVars[1] = 0.0
                             } else {
                                 melodyAudioManager.startPlayer(track: selectedMelodySound.name)
+                                
+                                selectedImageNames.melody = selectedMelodySound.imageName
+                                
+                                opacityAnimateVars[1] = 0.5
+                                
                             }
                         }
                     }
                 }
             }.padding(.horizontal, 15)
         }
     }
-
+    
     @ViewBuilder
     func SelectImage() -> some View {
-        Rectangle()
-            .frame(width: deviceFrame().exceptPaddingWidth, height: deviceFrame().exceptPaddingWidth, alignment: .center)
+        ZStack {
+            Rectangle()
+                .DeviceFrame()
+                .background(.gray)
+            
+            // Base
+            illustImage(imageName: selectedImageNames.base, animateVar: opacityAnimateVars[0])
+            
+            // Melody
+            illustImage(imageName: selectedImageNames.melody, animateVar: opacityAnimateVars[1])
+            
+            // Natural
+            illustImage(imageName: selectedImageNames.natural, animateVar: opacityAnimateVars[2])
+                
+        }
+    }
+    
+    @ViewBuilder
+    func illustImage(imageName: String, animateVar: Double) -> some View {
+        
+        
+        Image(imageName)
+            .resizable()
+            .DeviceFrame()
+            .opacity(animateVar)
+            .animation(.linear, value: animateVar)
+    }
+    
+    // MARK: imageName과 일러스트 이름이 다를 경우
+    func getIllustName(of imageName: String) -> String {
+        // base
+         switch imageName {
+            case "LongSun":
+              return "LongSun"
+            case "SpaceMid":
+              return "SpaceMid"
+            case "SpaceLow":
+              return "SpaceLow"
+            case "SpaceHigh":
+              return "SpaceHigh"
+            case "Oxygen":
+              return "Oxygen"
+            default:
+              return "LongSun"
+        }
+        // melody
+        // ..
+        // natural
+        // ..
     }
```

Comment:
switch 방법을 유지한다면
1.  case에 여러개를 한 번에 넣기
```suggestion
   func getIllustName(of imageName: String) -> String {
    // base
    switch imageName {
    case "LongSun", "SpaceMid", "SpaceLow", "SpaceHigh", "Oxygen":
        return imageName
    default:
        return "LongSun"
    }
}
```

Reply:
2. allCases 이용하기
 - static.swift에 있는  enum BaseAudioName에 프로토콜(String, CaseIterable)을 추가하기
```
 enum BaseAudioName:String, CaseIterable {
    case longSun
    case spaceMid
    case spaceLow
    case spaceHigh
    case oxygen
}
```
```suggestion
    // MARK: imageName과 일러스트 이름이 다를 경우
     func getIllustName(of imageName: String) -> String {
          // base
          switch imageName {
          case BaseAudioName.allCases.map({$0.rawValue}):
              return imageName
          default:
              return "LongSun"
          }
    }
```

Reply:
새롭게 배워갑니다! 근데 혹시 들어오는 imageName과 return될 일러스트 이름이 다른 경우에도 적용이 되는 것일까요?

Reply:
다르면 다시 돌아가야 합니다